integration-tests/src/test/kotlin/spine/protodata/hello/SizeOptionTest.kt

```diff
@@ -0,0 +1,53 @@
+package spine.protodata.hello
+
+import io.spine.protodata.hello.Address
+import io.spine.protodata.hello.Contact
+import io.spine.protodata.hello.validateAddressCount
+import io.spine.protodata.hello.validateAddressLineCount
+import io.spine.protodata.hello.validateEmailCount
+import io.spine.protodata.hello.validatePhoneCount
+import org.junit.jupiter.api.Test
+
+/**
+ * Checks various use-cases on `size` option validation.
+ */
+class `Size option test should` {
+
+    @Test
+    fun `check several options within the same message type and sub-type`() {
```

Comment:
Could you add some negative tests here, in the same manner as this one?

I assume we should be expecting some (particular) `RuntimeException` in such cases.

Reply:
Negative cases added, I just assumed that we have negative cases in the model tests.